requirements/prod.txt

```diff
@@ -2,4 +2,7 @@ django>=3.1
 psycopg2
 pucas>=0.6
 django-split-settings
-requests
\ No newline at end of file
+requests
+django-taggit
+#next version of piffle
+git+https://github.com/princeton-cdh/piffle.git@feature/iiif_pres#egg=piffle
```

Comment:
Heads up: This doesn't exist anymore

Reply:
oh, thanks for pointing that out; I merged it the piffle branch, and I thought I updated the requirement for geniza (should be develop now), but either I didn't or I didn't push it yet...